src/legacy/core_plugins/kibana/public/discover/components/field_chooser/discover_field_search.tsx

```diff
@@ -97,7 +97,7 @@ export function DiscoverFieldSearch({
       >
         <FormattedMessage
           id="kbn.discover.fieldChooser.fieldFilterFacetButtonLabel"
-          defaultMessage="Fields filtered"
+          defaultMessage="Filter by type"
```

Comment:
dear @bhavyarm, that's the wording we talked about, do you think it's better this way?

Reply:
We previously had this wording in Lens, but it was requested to change by @gchaps - this is at least now consistent

Reply:
@kertal thanks for checking in. I discussed it with QA team too. It isn't clear to us at all that  *Fields filtered* means we can change the facets(categories) of the fields.  Its not indicating an action - but you can clearly do it on Aggregatable and searchable and type. But we couldn't arrive at something definite. Does adding a tooltip will help? In any case - *Filter* in discover is getting confusing. Because we have data filters and then this which isn't necessarily filtering on data. Thanks!

Reply:
But I definitely don't want to get stuck in on this

Reply:
I like the idea of adding a tooltip.

Reply:
I'm not sure a tooltip improves the clarity of this, since the behavior is pretty discoverable- you can click the button and see what filters are available. I would prefer not introducing different tooltips for Lens and Discover

Reply:
What do you think about this way? There's a `Search for fields` and a `Filter fields` action, plus we could add a tooltip elaborating the square block displaying the number of active field filters.
<img width="450" alt="Bildschirmfoto 2019-10-29 um 11 14 11" src="https://user-images.githubusercontent.com/463851/67758136-61aa2380-fa3d-11e9-8538-734128b233c8.png">